File: core/src/com/mirage/controller/Platform.java
package com.mirage.controller;

public class Platform {
    public static final byte DESKTOP = 0;
    public static final byte ANDROID = 1;
    public static final byte IOS = 2;
    public static byte TYPE = DESKTOP;
    /**
     * Путь до папки с ресурсами, для разных платформ он разный
     */
    public static String ASSETS_PATH = "";
}

File: core/src/com/mirage/controller/Platform.kt
package com.mirage.controller

object Platform {
    enum class Types {
        DESKTOP,
        ANDROID,
        IOS
    }

    /**
     * Платформа, на которой запущена игра
     */
    var TYPE = Types.DESKTOP
    /**
     * Путь до папки с ресурсами, для разных платформ он разный
     */
    var ASSETS_PATH = ""
}
